modules/ROOT/pages/deploying.adoc

```diff
@@ -0,0 +1,31 @@
+= Deploying Mule Runtime
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: deploy, deploying, cloudhub, on premises, on premise
+
+[NOTE]
+This section deals with deploying to standalone instances of the Mule Runtime. To take full advantage of managing and monitoring features, you can deploy to the same Mule runtime via the xref:runtime-manager::index.adoc[Runtime Manager]. See xref:runtime-manager::deployment-strategies.adoc[Deployment Strategies] for a deeper look at the different deployment alternatives offered by the platform.
+
+Mule standalone can run multiple applications. This means you can include the same name spaces within different applications and they neither collide nor share information.
+
+In effect, Mule is an app server. Mule checks for new apps (in the apps directory) and starts them.
+
+Mule also restarts apps if it detects that the apps have changed, meaning you can drop a revised version of an existing, running application into the /apps directory, Mule stops the existing app and restarts using the new files.
+
+If Mule is embedded in another container Mule Management Console cannot perform Autodiscovery or server restarts.
```

Comment:
Mule Management Console (MMC) is EOLd, please remove this.

Reply:
Removed.